src/metax_api/api/oaipmh/base/metax_oai_server.py

```diff
@@ -0,0 +1,113 @@
+import datetime
+
+from django.utils import timezone
+from django.conf import settings
+from oaipmh import common
+from oaipmh.common import ResumptionOAIPMH
+from oaipmh.error import IdDoesNotExistError
+
+from metax_api.models.catalog_record import CatalogRecord
+
+
+class MetaxOAIServer(ResumptionOAIPMH):
+
+    def _get_set_filter(self, set=None):
+        return [1, 2]
+
+    def _get_filtered_records(self, set, cursor, batch_size, from_=None, until=None):
+        query_set = None
+        if from_ and until:
+            query_set = CatalogRecord.objects.filter(date_modified__gt=from_, date_modified__lt=until)
```

Comment:
is it gt or gte? Likewise for the lt/lte.

Reply:
https://stackoverflow.com/a/10040165/1201945 looks like _lte and _gte. But one of the ends need to be just lt/gt, both cannot be "or equal"